scripts/patches/debug_ce.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
is it on any use if file is not intended to be called directly?

(I'm also wondering about the executable file permission)

Reply:
The missing shebang suggests the script is source-d. But it expects `$1` to be present. It's not sourced, it's invoked like this in `call_patches`:

    "$script" "$sq_home"

To disambiguate how it will be interpreted, it really should have a shebang.

The current invocation pattern requires the executable permission, and I think it's good like this. Note that patches are not limited to Bash, they could be something else too (awk, python, perl), with appropriate shebang.

Reply:
sourced => no shebang, not sourced => sheband

this makes sense.

However, all our sourced .sh files in `scripts/` have shebangs and are executable.

Reply:
It would be good remove the executable bit from the scripts that are intended to be sourced. The shebang might be still good to keep, to signal to the reader the required compatibility level. If it's not there, I will wonder if it's ok to use Bash syntax (sourced by a Bash script) or if I should stick to `/bin/sh` syntax (sourced by `/bin/sh` script).

Reply:
sounds good to me